### DISCUSSION TITLE
Pouvoir accéder aux données d'export de Matomo dans un logiciel type Metabase

### DISCUSSION THREAD

**Discussion by ketsapiwiq** [betagouv/cap#24] [Besoin]
### Besoin

```bash
Avoir une interface plus agréable et maniable que Matomo.
```

### Solutions envisageables

```bash
1. https://github.com/SocialGouv/matomo-postgres
2. Export to CSV from API
3. Plugin lien "export to CSV" ?
```

### Informations complémentaires

- [x] J'avais déjà entendu ce besoin au sein de la communauté beta.gouv
- [x] Je me porte volontaire pour participer à la mise en place de la future solution choisie

**Comment by ketsapiwiq** [comment]
Après rapide regard, effectivement la bonne voie semble être ce dépôt matomo-postgres qui s'appuie sur le plugin Live.
Mais pour le lien d'export à un CSV par rapport à un rapport existant, il me semble qu'on peut suivre ça et obtenir un lien queryable qui permet de retélécharger périodiquement un CSV via un lien et un token : https://fr.matomo.org/faq/faq_26727/

Exemple : https://stats.beta.gouv.fr/index.php?module=API&format=CSV&idSite=126&period=month&date=2024-07-02&method=Actions.getEntryPageUrls&expanded=1&translateColumnNames=1&language=fr&token_auth=ENTER_YOUR_TOKEN_AUTH_HERE&filter_limit=100

**Comment by revolunet** [comment]
Une autre option qui ne nécessite pas le plugin "Live" : https://github.com/betagouv/matomo-to-pg/

Me contacter si besoin de tester cette soluce;

**Comment by revolunet** [comment]
Voir aussi ce retex de l'équipe Nos Gestes climat : https://pad.numerique.gouv.fr/QwHA7R2HSrqNNtTZx2_LMg#
